torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -393,8 +456,15 @@ struct GraphFuser {
     }
     topological_index[graph->return_node()] = i++;
 
-    for(auto it = nodes.rbegin(); it != nodes.rend();) {
-      it = scanNode(*it);
+    // Run the pass until no changes are made.
```

Comment:
At some point, we should add some verbose reporting mode so we can see how many iterations we're doing. Good for debugging compiler perf problems.

Reply:
Also, it's worth putting the example from the PR here to explain why running until fixpoint is necessary.

Reply:
Good point, I forgot to include the example (it used to be there in the previous commit).